operations/README.md

```diff
@@ -169,12 +169,10 @@ Additional links to power on sub-procedures provided for reference. Refer to the
 
 - [Power On and Start the Management Kubernetes Cluster](power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md)
 - Power On Compute Cabinets
-  - [Power On Compute Cabinets](power_management/Power_On_Compute_Cabinets.md) using CAPMC
   - [Power On Compute Cabinets](power_management/Power_Control_Service/Power_On_Compute_Cabinets.md) using PCS
 - [Power On the External Lustre File System](power_management/Power_On_the_External_Lustre_File_System.md)
 - [Power On and Boot Compute and User Access Nodes](power_management/Power_On_and_Boot_Compute_Nodes_and_User_Access_Nodes.md)
 - Recover from a Liquid Cooled Cabinet EPO Event
-  - [Recover from a Liquid Cooled Cabinet EPO Event](power_management/Recover_from_a_Liquid_Cooled_Cabinet_EPO_Event.md) using CAPMC
   - [Recover from a Liquid Cooled Cabinet EPO Event](power_management/Power_Control_Service/Recover_from_a_Liquid_Cooled_Cabinet_EPO_Event.md) using PCS
 
 ## Power management
```

Comment:
There are still three references to CAPMC in this section.  Are those supposed to remain?

Reply:
Yes, there are still refs all over for CAPMC since the service is still running. The references stay to allow admins to compare and contrast the different services way of handling operations.

I have hopefully changed all of the examples to use PCS. If not, then those would need to be updated.

Reply:
10-4